release_notes/ocp-4-19-release-notes.adoc

```diff
@@ -1136,7 +1099,7 @@ With this release, Cloud Credential Operator pods now deploy with the `readOnlyR
 [id="ocp-4-19-notable-technical-changes-loopback-cert_{context}"]
 === Extended loopback certificate validity to three years for kube-apiserver
 
-Previously, in-memory loopback certificates in kube-apiserver were issued for one year. This meant that if the kube-apiserver ran without rollouts or restarts, the kube-apiserver would stop functioning after one year. In {product-title} {product-version} the in-memory loopback certificates are extended so the kube-apiserver can run for three years without interruption.
+Previously, the self-signed loopback certificate for the Kubernetes API Server expired after one year. With this release, the expiration date of the certificate is extended to three years.
```

Comment:
I agreed with @dfitzmau to include this RN in his PR instead of mine: https://github.com/openshift/openshift-docs/pull/94749

LGTM is in one of the clones of the Jira issues. See: https://issues.redhat.com/browse/OCPBUGS-56835?focusedId=27401138&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-27401138

I'll close my PR.